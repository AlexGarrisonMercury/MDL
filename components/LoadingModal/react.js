import React from 'react'
import PropTypes from 'prop-types'
import classNames from 'classnames/bind'
import styles from './style.module.scss'
import { React as ModalBackdrop } from 'components/ModalBackdrop'
import AnimatedMasoniteLogo  from 'components/AnimatedMasoniteLogo'

const cx = classNames.bind(styles)

const LoadingModal = () => (

   <ModalBackdrop>
    <AnimatedMasoniteLogo centered = {false} black={true} />
  </ModalBackdrop>
 
)

export default LoadingModal
