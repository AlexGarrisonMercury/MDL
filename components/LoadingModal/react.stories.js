import React from 'react'
import { storiesOf } from '@storybook/react'
import LoadingModal from './react'
import Readme from './react.README.md'

storiesOf('LoadingModal', module)
  .addParameters({ readme: { sidebar: Readme } })
  .add('default', () => (
    <LoadingModal>
    
    </LoadingModal>
  ))
