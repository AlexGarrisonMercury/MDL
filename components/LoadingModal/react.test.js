import { shallow } from 'enzyme'
import React from 'react'
import LoadingModal from './react'

describe('LoadingModal', () => {
  it('should render without crashing', () => {
    expect(
      shallow(
        <LoadingModal>
        </LoadingModal>,
      ),
    ).toMatchSnapshot()
  })
})
