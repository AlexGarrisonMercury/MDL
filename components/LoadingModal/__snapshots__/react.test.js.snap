// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalBackdrop should render without crashing 1`] = `
<div
  className="modal-backdrop"
>
  <div
    className="modal-backdrop__content"
  >
    <div>
      Test
    </div>
  </div>
</div>
`;
